docs/include/mkdocs-macros/main.py

```diff
@@ -0,0 +1,63 @@
+def define_env(env):
```

Comment:
Maybe a small explanation as module docstring about what these things do and how others can use them in the documentation?

Reply:
Done.